index.html

```diff
@@ -320,7 +320,7 @@ <h2>
       </p>
         <ul>
           <li>
-URL scheme identifier (<code>did</code>)
+URI scheme identifier (<code>did</code>)
```

Comment:
This is not a typo, see this comment:

https://github.com/w3c/did-core/pull/351#issuecomment-665055361